engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -242,13 +242,15 @@ private void initRenderGraph() {
         renderGraph.addNode(worldReflectionNode, "worldReflectionNode");
 
         // sky rendering
-        FBOConfig reflectedRefractedBufferConfig = new FBOConfig(new ResourceUrn("engine:sceneReflectiveRefractive"), FULL_SCALE, FBO.Type.HDR).useNormalBuffer();
+        FBOConfig reflectedRefractedBufferConfig = new FBOConfig(RefractiveReflectiveBlocksNode.REFRACTIVE_REFLECTIVE, FULL_SCALE, FBO.Type.HDR).useNormalBuffer();
```

Comment:
Hmmm. Let's leave it as it is for the time being then. I suspect we'll eventually have to deal with all those instances from a broader perspective anyway.